### PR TITLE
19 hotfix header

### PR DESCRIPTION
**Pull request by sachie5**
Created an optional dropdown list to appear on header showing list of staff using mapping of staff data.

Header now has an additional onChange Prop for functionality with dropdown list.

### CODE REVIEW

File: src/components/Header/Header.scss [hunk 1]
```diff
@@ -27,24 +28,35 @@
...
   }
 }
 
 @media screen and (min-width: $desktop-breakpoint) {
   .header {
     background-color: $clr-white;
     display: flex;
-    flex-direction: row;
+    flex-wrap: nowrap;
+    box-shadow: 0px 0.4rem 0.8rem 0px $clr-accent-grey;
```
Comment (mh-fung) [inline src/components/Header/Header.scss]: Can u help to change it from px to rem?

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -17,9 +17,10 @@ const Home = ({ variant }: HomeProps) => {
         <Header
           title="Home"
           variant={variant}
-          buttonOption={false}
+          buttonOption={true}
```
Comment (mh-fung) [inline src/pages/Home/Home.tsx]: This should be set as false for this Home component

**Review by mh-fung — CHANGES_REQUESTED** [review]
All good, except the bits mentioned in the sections

**Review by mh-fung — APPROVED** [review]
Great code. Thanks

**Review by fihockey — APPROVED** [review]

**Pull request merged**